tagstudio/src/core/palette.py

```diff
@@ -283,6 +283,45 @@ class ColorType(IntEnum):
     },
 }
 
+_UI_COLORS: dict = {
+    "": {
+        ColorType.PRIMARY: "#333333",
+        ColorType.BORDER: "#555555",
+        ColorType.LIGHT_ACCENT: "#FFFFFF",
+        ColorType.DARK_ACCENT: "#1e1e1e",
+    },
+    "red": {
```

Comment:
how about using an enum for the keys instead of string. Then in the places where these variables are used, it would be obvious where that value comes from, and that it's not just some random string. Something like this:


```python
class UIColors(enum.StrEnum):
    DEFAULT = ""
    RED = "RED"
    GREEN = "GREEN"
    ...
```

and then


```suggestion
    UIColors.RED: {
```